core/src/test/java/org/apache/iceberg/TestSnapshotManager.java

```diff
@@ -427,6 +427,26 @@ public void testReplaceBranchNonExistingSourceFails() {
         .hasMessage("Ref does not exist: non-existing");
   }
 
+  @Test
+  public void testFastForwardBranchNonExistingBranchToUpdateFails() {
```

Comment:
minor: the `To` in the name implies that the `to` branch is the focus of this test, whereas it's actually the `from` branch

Reply:
Good catch yeah this was a copy paste error, I updated.